### PR TITLE
Dashboard - homepage

### LINKED ISSUE #137: backoffice ui: dashboard (homepage)
- [x] check requirements
- [x] create mockup (?)

### PR DESCRIPTION
**Pull request by kpsherva**
( closes #137 )
Dashboard View

### CODE REVIEW

File: invenio_app_ils/ui/src/invenio_app_ils/common/urls.js [hunk 1]
```diff
@@ -13,6 +13,9 @@ export const BackOfficeURLS = {
   loanDetails: `${BackOfficeBase}/loans/:loanPid`,
   patronDetails: `${BackOfficeBase}/users/:userPid`,
   locationList: `${BackOfficeBase}/locations`,
+  // TODO after editor create link estabilished
+  newLoan: `${BackOfficeBase}/loans/create`,
```
Comment (ntarocco) [inline invenio_app_ils/ui/src/invenio_app_ils/common/urls.js:18]: can you please change this to something like:
```
loanRequest: `${BackOfficeBase}/circulation/loans/request`,
loanCheckout: `${BackOfficeBase}/circulation/loans/checkout`,
```
and then we need to fix backend `create` -> `checkout`

**Review by ntarocco — DISMISSED** [review]
The popup, it is shown only on hover... I would really go for a specific `info` or `help` icon instead, and then on hover show the popup

File: invenio_app_ils/ui/src/invenio_app_ils/common/api/documents/serializer.js [hunk 2]
```diff
@@ -12,6 +12,19 @@ function serializeResponse(hit) {
   return result;
 }
 
+function toTableView(document) {
```
[the remark this answers is not in the document]
Comment (kpsherva) [reply invenio_app_ils/ui/src/invenio_app_ils/common/api/documents/serializer.js]: is it good f.e. formatter?

File: invenio_app_ils/ui/src/invenio_app_ils/common/api/loans/loan.js [hunk 3]
```diff
@@ -51,54 +59,87 @@ class QueryBuilder {
...
-    if (typeof state === 'undefined' || state === '') {
+    if (!state || _isEmpty(state)) {
       throw TypeError('state argument missing');
     }
-    this.stateQuery.push(`state:${QueryBuilder.paramToQuery(state)}`);
+    this.stateQuery.push(`state:${prepareSumQuery(state)}`);
+    return this;
+  }
+
+  overdue() {
```
[the remark this answers is not in the document]
Comment (kpsherva) [reply invenio_app_ils/ui/src/invenio_app_ils/common/api/loans/loan.js:104]: I thought about it but `with`  suggest filling the param of the method, and there is none in this case

**Pull request merged**